GitUI/UserControls/BranchComboBox.cs

```diff
@@ -19,6 +20,11 @@ public BranchComboBox()
             branches.DisplayMember = nameof(IGitRef.Name);
         }
 
+        [Browsable(true)]
+        [Category("Action")]
+        [Description("Invoked when branch selection changed")]
```

Comment:
```suggestion
        [Description("Occurs whenever the branch selection has changed.")]
```

Please add the same to the xml-docs.